client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -159,7 +159,10 @@ class OrderFulfillment extends Component {
 			'is-placeholder': wcsEnabled && ! labelsLoaded,
 		} );
 
-		if ( ! isShippable && ( ! wcsEnabled || ( labelsLoaded && hideLabels ) ) ) {
+		if (
```

Comment:
This logic is getting out of hand. Would you mind trying to clarify this `if` (and the next one) either breaking it into multiple, separate conditions, or at least adding comments? For example, `if( 'cancelled' ) return null;` can be its own condition, no need to pile it with the other ones.